lib/compat/wordpress-6.3/template-parts-preloading.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Patches resources loaded by the block editor page
+ * to include Template Part posts.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Preloads requests needed for common Template Part posts
+ *
+ * @param array $preload_paths    Preload paths to be filtered.
```

Comment:
We're missing a `@param` for the `$context` argument

Reply:
✅